Gems/Atom/RPI/Code/Source/RPI.Public/RenderPipeline.cpp

```diff
@@ -439,6 +443,67 @@ namespace AZ
             }
         }
 
+        const Ptr<PassAttachment> RenderPipeline::GetPipelineAttachment(const Name& name) const
```

Comment:
Again, let's please change all 'Pipeline' usage to 'Global'.
Why it's clear why you chose it, naming is about clarity and Pipeline does not reflect the usage or meaning and in fact even switch between Pipeline and Global occasionally.  Let's simply stick to 'Global' everywhere.